app/javascript/dashboard/components-next/NewConversation/helpers/composeConversationHelper.js

```diff
@@ -80,6 +80,11 @@ export const getCapitalizedNameFromEmail = email => {
   return name.charAt(0).toUpperCase() + name.slice(1);
 };
 
+export const getNameFromPhoneNumber = phoneNumber => {
+  // Remove the '+' prefix if it exists
+  return phoneNumber.startsWith('+') ? phoneNumber.slice(1) : phoneNumber;
```

Comment:
This check is already done on line 203 in createNewContact, so it's kinda redundant. Minor nit-pick

Reply:
Done https://github.com/chatwoot/chatwoot/pull/10568/commits/038fd433ccefa2199b9e9cc0b314309904eaf1cc